build/build-mrt.yml

```diff
@@ -122,6 +121,30 @@ steps:
     PathtoPublish: $(Build.SourcesDirectory)/mrtcore.$(buildPlatform).$(buildConfiguration).binlog
     artifactName: binlogs
 
+# Copy and Publish the test binaries so we can run them in Helix later.
```

Comment:
do this after VSTest?

Reply:
sure, good idea. probably want to make sure the tests pass before creating any artifacts.